tests/parallel/mapping-volume/ParallelCube1To3.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<precice-configuration>
+  <solver-interface dimensions="3">
+    <data:scalar name="DataOne" />
+
+    <mesh name="MeshOne">
+      <use-data name="DataOne" />
+    </mesh>
+
+    <mesh name="MeshTwo">
+      <use-data name="DataOne" />
+    </mesh>
+
+    <participant name="SolverOne1_3">
```

Comment:
```suggestion
    <participant name="SolverOne">
```
In my opinion a bit much to encode the test name into the names here.

Reply:
It made debugging easier when exporting data (otherwise I ended up with different tests overwriting each other's output) and helped me find relevant logs from irrelevant ones, but I could replace them if you think that's better.

Reply:
My normal workflow is to only run one case when debugging. Then, you don't run into this problem. Are you aware of this trick?

```bash
mpirun -np 4 ./testprecice -x  -t "*/*/*/*/*/ReadWriteScalarDataFirstParticipantInitData"
```

Reply:
Ah, good to know ! I didn't master this so I stickted to `ctest -R Parallel`which was a bit too general